bootstrap/bootstrap.go

```diff
@@ -160,6 +160,22 @@ func (b *Bootstrap) Run(ctx context.Context) (exitCode int) {
 	if phaseErr == nil && includePhase(`command`) {
 		var commandErr error
 		phaseErr, commandErr = b.CommandPhase(ctx)
+		/*
+			Five possible states at this point:
+
+			Pre-command failed
+			Pre-command succeeded, command failed, post-command succeeded
+			Pre-command succeeded, command failed, post-command failed
```

Comment:
In this case the post-command `phaseError` will be reported and the command exit code _will not be reported_.

That may be a change we want to make, but this pull request does not change that behaviour at this time.